tests/python_dependencies/impacket/nmb.py

```diff
@@ -815,7 +815,7 @@ def recv_packet(self, timeout = None):
         # The next loop is a workaround for a bigger problem:
         # When data reaches higher layers, the lower headers are lost,
         # and with them, for example, the source IP. Hence, SMB users
-        # can't know where packets are comming from... we need a better
```

Comment:
`impacket` is vendored code and should remain untouched, please submit these to the upstream project.

Reply:
See conversation above